### ISSUE TITLE
starting ENSIME results in ENSIME: (Disconnected)

### ISSUE THREAD

**Issue by philipmac** [ensime/ensime-server#958]
Hi, 
working through the https://github.com/ensime/ensime-server/wiki/Quick-Start-Guide
got up to the starting ensime part, 
M-x ensime
asks which .ensime proj and offers to use <base>/.ensime 
says it's connected to ENSIME, speaking protocol 0.8.11, please wait...
but the bar is telling me ENSIME is d/c

--

if I close emacs, and try to start ensime again (as above, same proj file) it *appears* to work and offers a message something like Death to Null or Happy hacking etc, but again appears to be instantly disconnected. 

Any ideas what I'm doing wrong here? 
details : this is my .ensime file
https://gist.github.com/philipmac/0112c4a9058a89cce355

**Comment by fommil** [comment]
```
:scala-version "2.9.2"
```

might be a problem, it's been a while since I've checked that it still works (getting it working is part of the 1.0 roadmap, so this is a temporary glitch).

Can you check that it works for a scala 2.11 project?

**Comment by philipmac** [comment]
Hi Tom, 
- grabbed a copy of scala 2.11.4
- updated the project built.sbt to reflect this (ie scalaVersion := "2.11.4") 
- loaded emacs & M-x ensime
- same message of apparent success, but Disconnected message remains. 

updated proj .ensime file
https://gist.github.com/philipmac/fdefb6d54c69e460059b

Thanks for your help!

**Comment by fommil** [comment]
ok, then it must be something in your environment.

I don't know what you mean by "grabbing a version of scala" because updating `scalaVersion` in your `build.sbt` will do this for you.

Looks like you forgot the `sbt gen-ensime` step to regenerate your `.ensime`.

Can you share your project on github?

**Comment by philipmac** [comment]
Oops, I did actually sbt gen-ensime, but forgot to note it above; the .ensime file was regenerated. 

Regarding sbt getting a diff ver of Scala as a result of my updating the vName in build.sbt, I was not aware it did this. By grab I meant I went to scala-lang.org, got the relevant deb file, and dpkg -i it. (Out of curiousity I bumped the v number up a release and re ran gen-ensime.)

Regarding sharing the project, of course:

https://github.com/philipmac/ensime-test
